serverless.yml

```diff
@@ -97,15 +107,36 @@ functions:
 resources: # CloudFormation template syntax from here on.
   extensions: ${file(./sls-config-${opt:stage}.yml), null}
   Resources:
+
+    ase-findme-image-upload-bucket:
+      Type: AWS::S3::Bucket
+      Properties:
+        BucketName: ase-findme-image-upload-bucket
+        AccessControl: PublicRead
```

Comment:
I guess we should make this bucket private at some point, but lets worry about permissions and roles later

Reply:
👍